### DISCUSSION TITLE
How do we enable the registration flow, if end customer is not already there in the system.

### DISCUSSION THREAD

**Discussion by dhanan-np** [JanssenProject/jans#4722] [Q&A] [answered]
How do we enable option for Registration flow when customer who does not hold account with our portal. 
I have seen an option with OpenId-Connect, where it says we can invoke it using the request parameter [**prompt=create**](https://openid.net/specs/openid-connect-prompt-create-1_0.html#authz-endpoint-example-code) 
but somehow it is not working for me.

**Comment by ossdhaval** [comment]
Hey @dhanan-np, this feature is already tracked using [this issue](https://github.com/JanssenProject/jans/issues/2616). It will be soon aligned to a future release. Thanks for bringing this up though. Give it a 👍 .

**Reply by dhanan-np** [reply]
so we can currently use only the backend rest api for the registration. right?

**Reply by ossdhaval** [reply, marked as answer]
Not sure which REST API you are referring to but I think it is worth going through [this](https://docs.jans.io/v1.0.12/admin/planning/identity-management/) document and it'll give you an understanding of the Janssen Project's approach towards `Identity Management`.

**Comment by nynymike** [comment]
In the Person Authn Script, you can detect if a user is new (for example, from social login) and add the user. You don't need the `prompt=create` authn request param to do this--the script can detect that it's a new person.

**Reply by dhanan-np** [reply]
so you mean, that we check in the script and redirect the user to some custom created registration form?
